actions/pack_action.go

```diff
@@ -30,12 +36,42 @@ type PackAction struct {
 	File             string
 }
 
+func NewPackAction() *PackAction {
+	d := PackAction{}
+	// Use gz by default
+	d.Compression = "gz"
+
+	return &d
+}
+
+func (pf *PackAction) Verify(context *debos.DebosContext) error {
+	possibleTypes := []string{"gz", "bzip2", "xz", "none"}
+	for _, a := range possibleTypes {
+		if a == pf.Compression {
+			return nil
+		}
+	}
+
+	return fmt.Errorf("Option 'compression' has an unsupported type: `%s`. Possible types are %s.",
+		pf.Compression, strings.Join(possibleTypes, ", "))
+}
+
 func (pf *PackAction) Run(context *debos.DebosContext) error {
 	pf.LogStart()
 	outfile := path.Join(context.Artifactdir, pf.File)
 
+	var tarOpt = "cf"
+	tarOpts := map[string]string{
+		"gz":    "z",
+		"bzip2": "j",
+		"xz":    "J",
+		"none":  "",
```

Comment:
Can you make it so this is used inboth verify and Run. Rather then Verify having it's own array of valid compression options that needs to be manually kept in sync? Otherwise looks great thanks!

Reply:
Okay, done :slightly_smiling_face: